.github/workflows/ci.yml

```diff
@@ -93,7 +93,7 @@ jobs:
         if: runner.os == 'Windows' && steps.changed-files.outputs.only_changed != 'true'
         run: |
           echo "PLAYWRIGHT_BROWSERS_PATH=$HOME\.cache\playwright-bin" >> $env:GITHUB_ENV
-          $env:PLAYWRIGHT_VERSION="$(pnpm ls --depth 0 --json -w playwright-chromium | jq --raw-output '.[0].devDependencies[\"playwright-chromium\"].version')"
+          $env:PLAYWRIGHT_VERSION="$(pnpm ls --depth 0 --json -w playwright-chromium | jq --raw-output '.[0].devDependencies["playwright-chromium"].version')"
```

Comment:
Thanks for fixing this! The issue has been haunting us in CI for many weeks now.